lib/core/test/time_point_test.cpp

```diff
@@ -44,3 +45,11 @@ TEST_F(TimePointTest, inplace_arithmetics) {
   EXPECT_EQ(aux -= 1, t1);
   EXPECT_EQ(aux += 1, t2);
 }
+
+TEST(TimePointTypeTest, is_pod) {
```

Comment:
Is it actually POD or just trivial? I though we settled on the latter?

Reply:
I don't know. `std::is_trivial` seems to be equivalent here: If I add/remove the member init then the two return the same, so I cannot tell which one it really is?